src/commands/build.js

```diff
@@ -68,13 +73,26 @@ module.exports = {
       manifest !== undefined && manifest !== ''
         ? manifest
         : filesystem.resolve('subgraph.yaml')
+    networkFile =
+      networkFile !== undefined && networkFile !== ''
+        ? networkFile
+        : filesystem.resolve("networks.json")
 
     // Show help text if requested
     if (help) {
       print.info(HELP)
       return
     }
 
+    if (network && filesystem.exists(networkFile) !== "file") {
+      print.error(`Network file '${networkFile}' does not exists!`)
+      process.exit(1)
```

Comment:
You actually might want to do this instead:
```javascript
process.exitCode = 1
return
```

It gives the same result, but perhaps that helps better with `gluegun`? 🤔 
Not sure.

Reply:
I thought they are the same 🤔 But I guess I should stick to the current style anyways

Reply:
They probably are.